crates/runner/src/vm/instructions/mod.rs

```diff
@@ -0,0 +1,57 @@
+use stwo_prover::core::fields::{m31::M31, qm31::QM31};
```

Comment:
Maybe add an introduction comment on how instruction are encoded explaining why QM31 is used